File: cfgs/java_cfgs_foo/CEILING_IN_A_SORTED_ARRAY.java
Nodes
1;special;;;
2;statement;int i;14;
3;conditional;foo_1 <= foo_2[foo_3];15;
4;statement;return foo_1;15;
5;exit;;;
6;statement;i = foo_1;16;
7;conditional;foo_1 < foo_2;17;
8;conditional;foo_1[foo_2] == foo_3;19;
9;statement;return foo_1;19;
10;conditional;(foo_1[foo_2] < foo_3) && (foo_4[foo_5 + 1] >= foo_6);20;
11;statement;return foo_1 + 1;20;
12;statement;i++;18;
13;statement;return -1;22;
Edges
0;2;
1;3;
2;4;6;
3;5;
4;
5;7;
6;8;13;
7;9;10;
8;5;
9;11;12;
10;5;
11;7;
12;5;

File: cfgs/pythoncfgs/CEILING_IN_A_SORTED_ARRAY.py
Nodes
1;conditional;foo_1 <= foo_2[foo_3];8;
2;statement;foo_1 = foo_2;10;
3;conditional;fun_1(foo_1);11;
4;statement;return -1;16;
5;conditional;foo_1[foo_2] == foo_3;12;
6;conditional;foo_1[foo_2] < foo_3 and foo_4[foo_5 + 1] >= foo_6;14;
7;statement;return foo_1 + 1;15;
8;statement;return foo_1;13;
9;statement;return foo_1;9;
10;exit;;
Edges
0;2;9;
1;3;
2;4;5;
3;10;
4;6;8;
5;3;7;
6;10;
7;10;
8;10;
9;